### ISSUE TITLE
default keyboard shortcuts should respect platform standards

### ISSUE THREAD

**Issue by hayne** [Komodo/KomodoEdit#1665]
### Short Summary
The defaults for some keyboard shortcuts in Komodo Edit do not conform to what OS X users expect

### Steps to Reproduce
The keyboard shortcut for Find Next is F3 by default.
OS X users (and my fingers) expect that it is Command-G

The keyboard shortcut for Replace (in Edit menu) is Command-H by default.
But Command-H on OS X is the usual shortcut for hiding the whole application.
(And is listed as such in the application menu ("Komodo Edit") at top left.)
I don't have a suggestion as to what to do for the keyboard shortcut for Replace
- but it definitely should not be that which is used system-wide for hiding an application.

### Expected results
The default for Find Next should be Command-G on OS X
The keyboard shortcut Command-H should always hide the application.

### Platform Information
Komodo Edit 10.0.0
OS X 10.9.5

**Comment by cgchoffman** [comment]
Thanks @hayne.  Good points.  I'll have a look at these and find better alternatives.

**Comment by hayne** [comment]
<img width="328" alt="textwrangler_searchmenu" src="https://cloud.githubusercontent.com/assets/712769/15834463/a1beacd6-2bf9-11e6-8a0d-c53cf2859484.png">
<img width="575" alt="msword_findmenu" src="https://cloud.githubusercontent.com/assets/712769/15834470/a6e4e22a-2bf9-11e6-8487-0c52a9ffa172.png">

Above I have attached screenshots (on OS X) of the Search menu from TextWrangler (free version of BBEdit) and the Find menu of Microsoft Word (Office 2011 for Mac) - these show some possibilities for the various keyboard shortcuts

**Comment by cgchoffman** [comment]
https://github.com/Komodo/KomodoEdit/commit/81819fc581702619d5f0e2d96a41464854575f1b
https://github.com/Komodo/KomodoIDE/commit/0cc4356d2fcc710c6f581a6dda1ff7d467de3173

Explanation for changes made (note internally `cmd` = `meta` in Komodo):
I've followed the suggestions made by @hayne for *find next*, `Meta+G`
For replace I've used the legacy command as I can't think of a better one.

I had to change the debugging commands since they were using `meta+G. *Start debugging* and *start debugging but skip dialog* are `Meta+.` and `Meta+shift+>` respectively.

I also took the liberty to change (probably shouldn't have...waiting for posterior to be bitten) the step in, step out and step over bindings so that are in an intuitive position and all located under the right hand rather than various locations around the keyboard.
I had an arrow pad in mind, `i` being up (step out), `k` being down (step in), and `l` being right (step over).  They are all preceded by `Meta+Ctrl+`.
